lib/process-custom-blocks.js

```diff
@@ -0,0 +1,41 @@
+const { getVueJestConfig, getCustomTransformer } = require('./utils')
+const vueOptionsNamespace = require('./constants').vueOptionsNamespace
+
+function applyTransformer(
```

Comment:
Should we try to match the jest process api a bit more closer? That would be `applyTransform(blocks, filename, config, transformer, vueOptionsNamespace) I guess. Although maybe more awkard to call then - just thinking out loud

Reply:
I wasn't sure about this either. I think if there was no external context, like if jest process didn't exist, maybe an object-based api would be best, because there's a lot of optional params.

Reply:
I wasn't sure about this either. I think if there was no external context, like if jest process didn't exist, maybe an object-based api would be best, because there's a lot of optional params.

Reply:
Object based works well for me too - what do you think? I really like to use an object when there are more than 1-2 params, since it is easier to read.

If it's not too much trouble that change could go with that? I don't mind too much, though.

Reply:
Updating it for this